src/fsharp/service/service.fs

```diff
@@ -1122,7 +1131,15 @@ type TypeCheckInfo
             (fun () -> 
                 match GetDeclItemsForNamesAtPosition(ctok, None,namesOpt,None,None,line,lineStr,colAtEndOfNames,ResolveTypeNamesToCtors,ResolveOverloads.No,(fun() -> []),fun _ -> false) with
                 | None -> FSharpMethodGroup("",[| |])
-                | Some (items, denv, _, m) -> FSharpMethodGroup.Create(infoReader, m, denv, items |> List.map (fun x -> x.ItemWithInst)))
+                | Some (items, denv, _, m) -> 
+                    // GetDeclItemsForNamesAtPosition returns Items.Types and Item.CtorGroup for `new T(|)`, 
+                    // the Item.Types is not needed here as it duplicates (at best) parameterless ctor.
```

Comment:
I'm afraid for _some_ cases there are _only_ `Item.Types` where ctors expected, so removing them would break those cases... We need the tests to work again :(